open_spiel/python/jax/opponent_shaping_jax_test.py

```diff
@@ -0,0 +1,144 @@
+# Copyright 2019 DeepMind Technologies Limited
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Tests for open_spiel.python.jax.lola."""
+import typing
+from typing import Tuple
+
+import distrax
```

Comment:
Ah, I think we'll also need to add distrax to [this line](https://github.com/deepmind/open_spiel/blob/f7a43409f35921b7486ef76387e0947520add04f/open_spiel/scripts/python_extra_deps.sh#L27) for this to run on CI

Reply:
I just made the requested changes and pushed a commit.